frontend/packages/dev-console/src/components/topology/components/nodes/WorkloadNode.tsx

```diff
@@ -96,6 +97,15 @@ const WorkloadNode: React.FC<WorkloadNodeProps> = ({
                 </Decorator>
               </Tooltip>
             ),
+            eventWarning && (
+              <ConnectedMonitoringDecorator
+                key="monitoring"
+                x={cx + -radius + decoratorRadius * 0.7}
+                y={cy + -radius + decoratorRadius * 0.7}
```

Comment:
Can we write it like this?

```suggestion
                y={cy - radius + decoratorRadius * 0.7}
```